data/json/effects.json

```diff
@@ -1955,15 +1955,17 @@
   {
     "type": "effect_type",
     "id": "hypocalcemia",
```

Comment:
Didn't Venera3 explain that basically hypocalcemia shouldn't be a thing because it can't devlop in the time frame of the game? https://github.com/CleverRaven/Cataclysm-DDA/pull/45564#discussion_r548869480

Reply:
I can comment it out,

> 
> 
> Didn't Venera3 explain that basically hypocalcemia shouldn't be a thing because it can't devlop in the time frame of the game? [#45564 (comment)](https://github.com/CleverRaven/Cataclysm-DDA/pull/45564#discussion_r548869480)

it shouldn't be, your intake should make it not happen unless it happens over many years assuming you eat something with calcium in it every once in while, and the maximum extent for it in this PR, is very mild to medium muscle stiffness, so only low level deficiency, especially given calcium pills exist around the place.

The reason it still exists, but in a very unlikely state, is so drugs that may impact the body's metabolism can make it happen over a more drastic timespan, thus the player has more consequences for their abuse longer-term.

Reply:
300 days of absolutely no intake is astronomically unlikely in a normal situation, but I don't currently know enough about drugs or injuries to really have a take on how much they'd impact this.

Reply:
I specifically only implemented mild symptoms for these reasons as well.

Reply:
> 
> 
> B12: a healthy nonvegan adult has about 2000-5000 mcg of it stored in their liver (that's 500-1250 day's RDA) so the timeline could stand to be bumped up by a factor of two at least. Adding a max value to represent it being stored would fit well. Also, #47291 added support for effects influencing vitamins, so you could hook up a nice B12/Red blood cell interaction instead of fudging it with fatigue/strength reduction.
> A and Iron look fine.
> Scurvy's fatigue could use a safety `fatigue_max_val`, otherwise it looks good.

what do you suggest as the value?